src/new-client/src/app/site/[site]/details/edit-information-for-citizens/page.tsx

```diff
@@ -0,0 +1,37 @@
+import NhsPage from '@components/nhs-page';
+import { fetchPermissions, fetchSite } from '@services/appointmentsService';
+import { EditInformationForCitizensPage } from './edit-information-for-citizens-page';
+
+export type PageProps = {
+  params: {
+    site: string;
+  };
+};
+
+const Page = async ({ params }: PageProps) => {
+  const site = await fetchSite(params.site);
+  const siteMoniker = site?.name ?? `Site ${params.site}`;
```

Comment:
You can ditch the siteMoniker stuff now, since the result of `fetchSite` can be relied upon